### PR TITLE
Added the flag --no-resize for just cropping the image and saving it.

### PR DESCRIPTION
**Pull request by ASHWIN990**
**Added the flag --no-resize for just cropping the image and saving it.**

This issue i also faced, and wanted to work on that, some times the small res images get all pixelated when its resized, and saw an opened issue #57 then i think that was the time.

**Things Done**
* Added the flag --no-resize 
* The print status was giving incomplete data (not printing number of rejected images)
* Updated the "README.md"

**How it works**

Basically when you pass *--no-resize* it will stored as True and in both API and the cli version script will check if the flagg is passed or not if yes then it will check the condition and call the *cropper* class based on that.

**Things to do**

I have tested it  in all means but, i don't know how to write the automatic testing script, but you can do it, or you can teach me how to do that, it will be very helpfull and thoughfull of you if you do that..

### CODE REVIEW

**Comment by leblancfg** [comment]
That's wonderful, thanks for the PR @ASHWIN990!

In order to merge this, could you please:
1. Add a test that checks the output size of the image, one with the `--no-resize` flag, and one without?

Also, the test failure should be easy to fix. Make sure to test your code with `make check` before pushing your commits. 😜

File: autocrop/cli.py [hunk 1]
```diff
@@ -92,7 +92,16 @@ def main(
...
-    cropper = Cropper(width=fwidth, height=fheight, face_percent=facePercent)
+
+    """
+    Check if the no resize flag is provided or not.
+    """
+
+    if no_resize is True:
+        cropper = Cropper(width=fwidth, height=fheight, face_percent=facePercent, no_resize=True)
+    else:
+        cropper = Cropper(width=fwidth, height=fheight, face_percent=facePercent)
```
Comment (leblancfg) [inline autocrop/cli.py:103]:
```suggestion
    cropper = Cropper(width=fwidth, height=fheight, face_percent=facePercent, no_resize=no_resize)
```

File: autocrop/autocrop.py [hunk 2]
```diff
@@ -128,11 +128,13 @@ def __init__(
         face_percent=50,
         padding=None,
         fix_gamma=True,
+        no_resize=False,
```
Comment (leblancfg) [inline autocrop/autocrop.py:131]: Can we rename this attribute as `self.resize` in the `Cropper` class please? It's confusing to follow along with the double-negative 😄

File: autocrop/autocrop.py [hunk 3]
```diff
@@ -207,10 +209,15 @@ def crop(self, path_or_array):
...
-        )
+        """
+        Check if the no resize flag is given or not, if given,
+        it will not resize the image just crop it
+        """
+        if self.no_resize is not True:
+            # Resize
+            image = cv2.resize(
+                image, (self.width, self.height), interpolation=cv2.INTER_AREA
+            )
```
Comment (leblancfg) [inline autocrop/autocrop.py:220]: Just so I understand, have you implemented the feature yet @ASHWIN990 ?

Comment (ASHWIN990) [reply autocrop/autocrop.py:220]: Yes, the feature has been implemented, and i have manually checked the output image resolution, and they are as it should be.

The main issue is that, i don't know how to write the test, maybe u can help me with that, teach me most probably if possible...

Comment (leblancfg) [reply autocrop/autocrop.py:220]: Writing the test should be pretty straightforward, but I'm not sure how comfortable you might be with unit testing and pytest. If not, here are [some great resources](https://www.freecodecamp.org/news/an-introduction-to-testing-in-python/) to [get you started](https://jacobian.org/2016/nov/27/getting-started-with-pytest/). I also found [this DataCamp course](https://www.datacamp.com/courses/unit-testing-for-data-science-in-python#!) to be quite helpful. I can help you out of course, if you have specific questions.

Background: `Cropper.aspect_ratio` is calculated from the height and width variables.

Basically, you want two tests:
1. Running `Cropper.crop(width=100, height=100, resize=True)` outputs a numpy array of size `100, 100`,
2. Running `Cropper.crop(width=100, height=100, resize=False)` outputs a numpy array whose size matches some hardcoded values you'll have gotten by running it on one of the test images. It should be square (`width == height`).

**Comment by leblancfg** [comment]
@ASHWIN990 did you still intend on working on this? I will close it otherwise in a few weeks.

**Comment by leblancfg** [comment]
Closing this PR for now. The idea is great and I hope to add the feature at a later time.
